lib/src/dom/accelerometer.dart

```diff
@@ -30,11 +30,16 @@ extension AccelerometerExtension on Accelerometer {
 
 @JS('AccelerometerSensorOptions')
 @staticInterop
+@anonymous
 class AccelerometerSensorOptions extends SensorOptions {
-  external factory AccelerometerSensorOptions();
+  external factory AccelerometerSensorOptions(
+      {AccelerometerLocalCoordinateSystem referenceFrame = 'device'});
```

Comment:
As we know, this will be ignored with existing object literal semantics. In the future too, this will just make the default value explicit in the options object users will pass. As such, do you think there is value in having these defaults here?

Reply:
Let's leave them now for documentation. It's true they aren't really used, but that doesn't matter in this case because they are 'default' from the perspective of the IDL. If we ever support default values, it won't be harmful to pass these to the IDL, but we might want to make them comments at that point just because it could possibly be a bit more efficient.